src/lib/components/trash-removed.svelte

```diff
@@ -3,12 +3,14 @@
     export let text
 </script>
 
-<section class="box-1">
+<!-- was box-1 -->
+<section class="trash-removed-total-box">
```

Comment:
Er valt me op dat je bij  trash-removed-total-box hebt gekozen om -box op het einde te zetten, bij trash-removed-last-month is er geen -box op het einde. Wat mij betreft is het meest consequent als ze allemaal wel -box hebben of allemaal niet. :)

Reply:
Goed opgemerkt!